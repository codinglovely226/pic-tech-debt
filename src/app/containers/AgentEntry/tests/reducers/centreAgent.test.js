/* eslint-disable */

import agentsReducer from '../../reducers/centreAgent';

import {
  addAgentCentreAgent,
  updateAgentCentreAgent,
} from '../../actions/case';

describe('agentsReducer', () => {
  it('returns the initial state', () => {
    expect(agentsReducer(undefined, {})).toMatchSnapshot();
  });
  it('add agent centre agent', () => {
    expect(agentsReducer({}, addAgentCentreAgent(1, 1, 'Test Agent', '', '1', 'mg'))).toMatchSnapshot();
  });
  it('update agent centre agent', () => {
    expect(agentsReducer({}, updateAgentCentreAgent(1, 1, 1, 'Test Agent', 'Test URL', '1', 'mg'))).toMatchSnapshot();
  });
});
