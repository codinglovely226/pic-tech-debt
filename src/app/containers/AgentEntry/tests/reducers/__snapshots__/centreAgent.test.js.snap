exports[`agentsReducer add agent centre agent 1`] = `
Object {
  "name": "Test Agent",
  "quantity": "1",
  "units": "mg",
  "url": "",
}
`;

exports[`agentsReducer returns the initial state 1`] = `
Object {
  "name": "",
  "quantity": "",
  "units": "",
  "url": "",
}
`;

exports[`agentsReducer update agent centre agent 1`] = `
Object {
  "name": "Test Agent",
  "quantity": "1",
  "units": "mg",
  "url": "Test URL",
}
`;
